lib/widgets/action_sheet.dart

```diff
@@ -402,3 +405,70 @@ class ShareButton extends MessageActionSheetMenuItemButton {
     }
   }
 }
+
+class MarkAsUnreadButton extends MessageActionSheetMenuItemButton {
+  MarkAsUnreadButton({
+    super.key,
+    required super.message,
+    required super.messageListContext,
+  }) : narrow = MessageListPage.ancestorOf(messageListContext).narrow;
+
+  final Narrow narrow;
+
+  @override IconData get icon => Icons.mark_chat_unread_outlined;
```

Comment:
Looks like we got this from Material Design. Since we don't have a design for this, maybe we can confirm that with Vlad?